charts/kafka-connect/values.yaml

```diff
@@ -0,0 +1,13 @@
+## The image is built with the Dockerfile in /images/kafka-connect
+image:
+  repository: us.gcr.io/galoy-org/kafka-connect
+  digest: "sha256:536348d31fdc138012a93b5ad54761c45075952e7c8d31508e629215e000de18"
+
+## The pods from this namespace are allowed to access the Kafka Connect API
+smoketestNameSpace: "galoy-dev-smoketest"
```

Comment:
```suggestion
smoketestNamespace: "galoy-dev-smoketest"
```

Reply:
while working on the testflight found that disabling the NetworkPolicy in Kafka altogether is probably the best approach. It adds unnecessary complication and not being used in our other charts either.